### DISCUSSION TITLE
Nuclei Openapi Dast scan error

### DISCUSSION THREAD

**Discussion by mikemicky4321** [projectdiscovery/nuclei#5050] [General]
Hello Team,
We are using openapi 3.0.1 version. We are planning to use Nuclei as DAST scanner. I am getting below error. Could you please also make an example How to scan nuclei using Openid auth? It would be really great and helpful
**Could not generate requests from op: [openapi:RUNTIME] unsupported security scheme type (openIdConnect) found in openapi file** or 
**Could not generate requests from op: [openapi:RUNTIME] unsupported security scheme type (oauth2) found in openapi file**
![image](https://github.com/projectdiscovery/nuclei/assets/115986628/4055109d-b406-4e2b-8ba0-bbfa5f75521d)
![image](https://github.com/projectdiscovery/nuclei/assets/115986628/8a135313-bf46-43d7-8c67-2d320db89d38)

**Comment by vladdragomir** [comment]
Hello!

We're getting the same `Could not generate requests from op: [openapi:RUNTIME] unsupported security scheme type (oauth2) found in openapi file` error. 

Do you have it on the roadmap to add support for the oauth2 security scheme type? 

Thanks!

**Reply by ehsandeep** [reply]
https://github.com/projectdiscovery/nuclei/issues/5069

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
